library/src/test/java/com/getstream/sdk/chat/NewLineTest.java

```diff
@@ -0,0 +1,13 @@
+package com.getstream.sdk.chat;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class NewLineTest {
+    @org.junit.jupiter.api.Test
+    void newlineSimpleTest() {
+        String string = "\n\n\n .a. \n\n\n";
+        String result = string.replaceAll("^[\r\n]+|[\r\n]+$", "");
```

Comment:
you should test `getDeletedOrMentionedText` not the regex ;)

Reply:
Oh, Yes. That was my local test. :)